diff --git a/package.json b/package.json
index 8861f13..0e85c95 100644
--- a/package.json
+++ b/package.json
@@ -4,6 +4,7 @@
   "description": "Tracker for The Skyward Sword Randomizer",
   "license": "GPL-3.0-or-later",
   "private": true,
+  "homepage": "/SS-Randomizer-Tracker/",
   "dependencies": {
     "@react-hook/resize-observer": "^1.2.6",
     "@reduxjs/toolkit": "^2.0.0",
diff --git a/src/App.tsx b/src/App.tsx
index e0d63c8..daaec68 100644
--- a/src/App.tsx
+++ b/src/App.tsx
@@ -4,7 +4,7 @@ import Shell from './Shell';
 
 function App() {
     return (
-        <Router>
+        <Router basename="/SS-Randomizer-Tracker">
             <Routes>
                 <Route path="/" element={<Shell />} />
                 <Route path="/acknowledgement" element={<FullAcknowledgement />} />
