diff --git a/package.json b/package.json
index 8861f13..0e85c95 100644
--- a/package.json
+++ b/package.json
@@ -4,6 +4,7 @@
   "description": "Tracker for The Skyward Sword Randomizer",
   "license": "GPL-3.0-or-later",
   "private": true,
+  "homepage": "/SS-Randomizer-Tracker/",
   "dependencies": {
     "@react-hook/resize-observer": "^1.2.6",
     "@reduxjs/toolkit": "^2.0.0",
diff --git a/src/App.tsx b/src/App.tsx
index 73cbf10..9889455 100644
--- a/src/App.tsx
+++ b/src/App.tsx
@@ -16,7 +16,7 @@ function App() {
     }, [colorScheme]);
 
     return (
-        <Router>
+        <Router basename="/SS-Randomizer-Tracker">
             <Routes>
                 <Route path="/" element={<Options />} />
                 <Route path="/tracker" element={<Tracker />} />
